### DISCUSSION TITLE
Plotting on Buissness-PC and Farming from HDD

### DISCUSSION THREAD

**Discussion by Hierbajo** [Chia-Network/chia-blockchain#977] [Q&A] [answered]
Hi!

I'm interested in joining your "Chia blockchain party" in (hopefully) March.

Thinking of plotting on a medium fast business PC like an Fujitsu Esprimo with an i5 9400; 16GB DDR4 RAM and a 512GB NVMe SSD.
- Can a machine like this plot in feasible time? Under lets say 15h or so?
- If yes can i pack the housing with cheaper (but quality) HDD's and farm from those?

Happy about anybodies short opinion!

Thanks
Hans

**Comment by hoffmang9** [comment, marked as answer]
You should be able to get in the 9-11 hour range with that. Here are some examples to look at - https://github.com/Chia-Network/chia-blockchain/wiki/Reference-Plotting-Hardware

You can see how people are going about farming - including packing a desktop case - here - https://github.com/Chia-Network/chia-blockchain/wiki/Reference-Farming-Hardware

Zoom on plotting today (2/19 in about 30 minutes - https://www.chia.net/2021/02/16/plotting-experts.html

-Gene

**Reply by Hierbajo** [reply]
Thank''s Gene for the quick answer! 

I already had a look at those wiki entries and found it informative but could not answer the question whether HDD's are feasible, fast enough, when moving plots from the temp drive to the "farming drive"...

Thanks for the invitation, maybe you guys record it. The day was too long already! 

Best wishes,
Hans
